src/Randomizer.CrossPlatform/Views/TrackerWindow.axaml.cs

```diff
@@ -103,8 +120,8 @@ private void LayoutMenuItemOnClick(object? sender, RoutedEventArgs e)
     }
 
     protected override string RestoreFilePath => Path.Combine(Directories.AppDataFolder, "Windows", "tracker-window.json");
-    protected override int DefaultWidth => 800;
-    protected override int DefualtHeight => 600;
+    protected override int DefaultWidth => 350;
+    protected override int DefualtHeight => 300;
```

Comment:
Defualt?

Reply:
Can't believe I goofed on that... It's in a small nuget chain. I'll fix in a separate PR.